src/repository/Repository.ts

```diff
@@ -24,13 +25,13 @@ class Repository {
   public async saveRewardEvent(rewarder: User, rewardees: User[], channelID: string, karma = 1) {
     await Promise.all(
       rewardees.map(async (rewardee) => {
-        const eventInstance = new RewardEvent();
-        eventInstance.rewarder = rewarder.id;
-        eventInstance.timestamp = new Date();
-        eventInstance.karma = karma;
-        eventInstance.channel = channelID;
-        eventInstance.rewardee = rewardee.id;
-        await RewardEvent.save(eventInstance);
+        await RewardEvent.create({
+          channel: channelID,
+          rewardee: rewardee.id,
+          karma,
+          timestamp: new Date(),
```

Comment:
Use `Date.now()` and not `new Date()`. Former is a number in milliseconds (or microsends?), while latter is a date object that'd be stringified to some equivalent string representation.

Reply:
https://github.com/indiainvestments/reward-bot/blob/585e5aeef578144b06e5536f66a4b33b828c4a60/src/entity/RewardEvent.ts#L9

ts type is `Date` and column type is `time with timezone`, so I don't think it would accept only epoch millis